core/domain/protofiles/training_job_data.proto

```diff
@@ -0,0 +1,33 @@
+// coding: utf-8
+//
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+import "core/domain/protofiles/text_classifier.proto";
+
+message TrainingJobData {
+  message JobResult {
+    string job_id = 1;
```

Comment:
I suggest adding comments to every field of all messages that you add. It's not clear to me what these values are meant to contain, and the added documentation would be helpful.

It's also helpful to add documentation to the messages themselves to provide context on how & where they're to be used.

Reply:
Added documentation.